src/components/common/UserArticleCardComponent.js

```diff
@@ -0,0 +1,78 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import Card from 'react-bootstrap/Card';
+import BadgeComponent from './BadgeComponent';
+import LikeComponent from './LikeComponent';
+import DislikeComponent from './DislikeComponent';
+import CommentComponent from './CommentComponent';
+import TimerComponent from './TimerComponent';
+import BookmarkComponent from './BookmarkComponent';
+
+const UserArticleCard = ({
+  img,
+  articleTitle,
+  tags,
+  likeCount,
+  dislikeCount,
+  commentCount,
+  timeCount
+}) => (
+  <Fragment>
+<Card style={{ width: '70rem', height: 'auto' }} className = 'mb-3 article-card'>
```

Comment:
this is how the cards were used in the react bootstrap docs

Reply:
Ok...

Reply:
Have you tried styling this otherwise? 🤔
Meanwhile, you can actually declare a variable object at the top of the file and pass it in e.g.
`const cardStyle = { width: '70rem', height: 'auto' }` and then `<Card style={cardStyle}`.

Reply:
Updated